webpack.config.js

```diff
@@ -16,10 +17,23 @@ var config = require( './server/config' ),
  * Internal variables
  */
 var CALYPSO_ENV = process.env.CALYPSO_ENV || 'development',
+	AUTOOPEN = process.env.AUTOOPEN || 'false',
 	PORT = process.env.PORT || 3000,
 	jsLoader,
 	webpackConfig;
 
+function OpenCalypsoInBrowser() {
+	this.isFirstBuild = true;
+}
+OpenCalypsoInBrowser.prototype.apply = function( compiler ) {
+	compiler.plugin( 'done', function() {
+		if ( this.isFirstBuild ) {
+			open( 'http://calypso.localhost:3000' );
```

Comment:
The port here at a minimum should be interpolated with the `PORT` variable as it is in [line 102](https://github.com/Automattic/wp-calypso/pull/734/files#diff-11e9f7f953edc64ba14b0cc350ae7b9dL102)